index.html

```diff
@@ -143,39 +152,41 @@ <h3 class="connect"><a href="">LET’S CONNECT</a></h3>
 				<li><a href=""><img src="img/chill.svg" alt=""></a></li>
 			</ul>
 			<button class="btn">Get my resume</button>
-	</div>
-	<div class="lang1">
-		<h3>Languages</h3>
-		<img src="/img/Union.svg" alt="">
-	</div>
-	<ul class="drop1">
-		<li>
-			<img src="/img/js_Ellipse 1.svg" alt="">
-			<p>Javascript</p>
-		</li>
-		<li>
-			<img src="/img/html_Ellipse 1.svg" alt="">
-			<p>HTML</p>
-		</li>
-		<li>
-			<img src="/img/css-Ellipse 1.svg" alt="">
-			<p>css</p>
-		</li>
-	</ul>
-	<div class="lang2">
-		<h3>Frameworks</h3>
-		<img src="/img/Unionarrdwn.svg" alt="">
-	</div>
-	<hr>
-	<div class="lang3">
-		<h3>skills</h3>
-		<img src="/img/Unionarrdwn.svg" alt="">
-	</div>
-	<hr>
+	    </div>
+		<div class="about_info">
+			<div class="lang1">
+				<h3>Languages</h3>
+				<img src="/img/Union.svg" alt="">
+			</div>
+			<ul class="drop1">
+				<li>
```

Comment:
- [ ] I like the way you set the about me section :+1: :100:, but there are some things that you could do to improve it:
  - Kindly place the technologies ("JavaScript", "HTML" and "CSS") in a row instead of a column to match as much as possible with the Figma template :+1: 
  - As mentioned before, kindly check the color of the "LET'S CONNECT" element because I think is a little bit lighter :+1: 
  - While you did a wonderful job styling your project :+1: , as a project requirement you should make the language, skills, and framework sections to be dropdown menus and add 5 tags per row listing the technologies :+1:. You can check this [video](https://www.youtube.com/watch?v=eKo1NV1qxbc) where it is done using only HTML and CSS.

**Your design**

![Screenshot from 2023-01-19 20-43-23](https://user-images.githubusercontent.com/105117832/213602909-b2826184-614d-4f6d-88ec-1b9a805491c5.png)

**Figma template**

![Screenshot from 2023-01-19 20-43-00](https://user-images.githubusercontent.com/105117832/213602942-e83ccd9b-e3d2-4ec7-bb80-515644d0e82c.png)